### PR TITLE
[CVEcheckerTask] get CVSS score, vector and severity from NVD

### PR DESCRIPTION
**Pull request by jpopelka**
The OSS Index does not provide CVSS score/vector/severity, so we need to get it from different source, which is easy once we know the CVE id.

### CODE REVIEW

**Comment by jpopelka** [comment]
[tests](https://ci.centos.org/job/devtools-fabric8-analytics-worker-fabric8-analytics/61/) are green.
Anybody for review ? @pkajaba @fridex

File: f8a_worker/workers/CVEchecker.py [hunk 1]
```diff
@@ -19,16 +21,46 @@ class CVEcheckerTask(BaseTask):
     schema_ref = SchemaRef(_analysis_name, '3-0-0')
 
     @staticmethod
+    def get_cve_impact(id_):
+        score = 0
+        vector = ''
+        severity = ''
+        if id_:
+            url = "https://nvd.nist.gov/vuln/detail/{id_}".format(id_=id_)
```
Comment (pkajaba) [inline f8a_worker/workers/CVEchecker.py]: This is just nitpicking, but you don't have to define variable here and put into requests.get instead

File: f8a_worker/workers/CVEchecker.py [hunk 2]
```diff
@@ -19,16 +21,46 @@ class CVEcheckerTask(BaseTask):
...
+            response = requests.get(url)
+            if response.status_code == 200:
+                score_v2, score_v3 = 0, 0
+                severity_v2, severity_v3, vector_v2, vector_v3 = '', '', '', ''
```
Comment (pkajaba) [inline f8a_worker/workers/CVEchecker.py]: would not this be nicer? 

`severity_v2 = severity_v3 = vector_v2 = vector_v3 = ''`

File: f8a_worker/workers/CVEchecker.py [hunk 3]
```diff
@@ -19,16 +21,46 @@ class CVEcheckerTask(BaseTask):
...
+                        score_v3 = float(tag.text.strip())
+                        severity_v3 = tag.find_next().text.lower()
+                    elif tag.attrs.get('data-testid') == 'vuln-cvssv3-vector-link':
+                        vector_v3 = tag.text.strip()
+                    elif tag.attrs.get('data-testid') == 'vuln-cvssv2-base-score-link':
+                        score_v2 = float(tag.text.strip())
+                        severity_v2 = tag.find_next().text.lower()
+                    elif tag.attrs.get('data-testid') == 'vuln-cvssv2-vector-link':
+                        vector_v2 = tag.text.strip().lstrip('(').rstrip(')')
+                score = score_v3 or score_v2
```
Comment (pkajaba) [inline f8a_worker/workers/CVEchecker.py]: comment here might be useful that we prefer v3 scores

**Comment by jpopelka** [comment]
Thanks, improved, PTAL

**Comment by pkajaba** [comment]
Thanks for changes, let's wait for test and proceed to merge.

**Pull request merged**